packages/workflow/src/activities/generateOrAssignContentType.ts

```diff
@@ -38,25 +38,19 @@ export async function guessOrCreateDocumentType(payload: DSLActivityExecutionPay
     log.info("SelectDocumentType for object: " + objectId, { payload });
 
     const object = await client.objects.retrieve(objectId, "+text");
+
+    //Expects object.type to be null on first ingestion of content
+    //User can set type manually via "Change Content type" via change-type.ts
```

Comment:
```suggestion
    //User can set type manually via "Change Content type" via the Composable UI
```

Reply:
```suggestion
    //User initiated Content Type change via the Composable UI,
    //sets object.type to null when they let Composable choose for them.
    //sets object.type to chosen type (thus non-null) when user picks a type.
```

This is closer to the intended purpose of the comment, while removing reference to change-type.ts.